docs/Evaluation.md

```diff
@@ -1,18 +1,86 @@
 # Evaluation 
 
-Independent analysis of whether a model can generalize from training data to new areas is critical for creating a robust workflow. 
-We stress that evaluation data must be different from training data, as neural networks have millions of parameters and can easily memorize thousands of samples. Therefore, while it would be rather easy to tune the model to get extremely high scores on the training data, it would fail when exposed to new images.
+ We stress that evaluation data must be different from training data, as neural networks have millions of parameters and can easily memorize thousands of samples. Avoid random train-test splits, try to create test datasets that mimic downstream tasks. If you are predicting among temporal surveys or across imaging platforms, your train-test data should reflect these partitions. Random sampling is almost never the right choice, biological data often has high spatial, temporal or taxonomic correlation that makes it easier for your model to generalize, but will fail when pushed into new situations.
+
+DeepForest provides several evaluation metrics. There is no one size fits all evaluation approach and the user needs to consider is the evaluation metric best fits the task. There is significant information online evaluation of object detection networks. Our philosophy is to provide a user a range of statistics and visualizations. Always visualize results and trust your judgement. Never be guided by a single metric.
+
+### Further Reading
+
+[MeanAveragePrecision in torchmetrics](https://medium.com/data-science-at-microsoft/how-to-smoothly-integrate-meanaverageprecision-into-your-training-loop-using-torchmetrics-7d6f2ce0a2b3)
+
+[A general explanation of the mAP metric](https://jonathan-hui.medium.com/map-mean-average-precision-for-object-detection-45c121a31173)
+
+[Comparing Object Detection Models](https://www.comet.com/site/blog/compare-object-detection-models-from-torchvision/)
+
+## Average Intersection over Union
+
+DeepForest modules use torchmetric's [IntersectionOverUnion](https://torchmetrics.readthedocs.io/en/stable/detection/intersection_over_union.html) metric. This calculates the average overlap between predictions and ground truth boxes. This can be considered a general indicator of model performance, but not sufficient on its own for model evaluation. There are lots of reasons predictions might overlap with ground truth, for example consider a model that covered an entire image with boxes. This would have high IoU, but low value to the user. 
```

Comment:
This calculates the average overlap between predictions and ground truth boxes. This can be considered a general indicator of model performance but is not sufficient on its own for model evaluation. There are lots of reasons predictions might overlap with ground truth; for example, consider a model that covers an entire image with boxes. This would have a high IoU but a low value.